.circleci/run.sh

```diff
@@ -183,6 +183,14 @@ check_d_builder()
     # TODO: add support for 32-bit builds
     ./src/build.d MODEL=64
     ./generated/linux/release/64/dmd --version | grep -v "dirty"
+
+    echo "Test CXX header generation"
```

Comment:
Please put this into a different/own function/step, it's not testing `build.d`.

Reply:
I did this initially, but I had to duplicate most of this function, hence I chose to avoid the duplication. Anyhow, I can see your motivation and I moved it back into a new function.